### PR TITLE
PM-13908 fixing copy on step2 and step3 and making vault url dynamic

### PR DESCRIPTION
**Pull request by dseverns-livefront**
## 🎟️ Tracking
https://bitwarden.atlassian.net/browse/PM-13908

## 📔 Objective
- make the environment set the url used for instruction on step 2.
- fix the copy on the help text.

## 📸 Screenshots
image

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed
  issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: app/src/main/res/values-da-rDK/strings.xml [hunk 1]
```diff
@@ -1055,7 +1055,7 @@ Skift til denne konto?</string>
   <string name="then_done_highlight">dernæst Færdig</string>
   <string name="for_your_security_be_sure_to_delete_your_saved_password_file">Sørg af sikkerhedshensyn for at slette den gemte adgangskodefil.</string>
   <string name="delete_your_saved_password_file">slet den gemte adgangskodefil.</string>
-  <string name="need_help_checkout_out_import_help">Behov for hjælp? Tjek import-hjælp ud.</string>
+  <string name="need_help_check_out_import_help">Behov for hjælp? Tjek import-hjælp ud.</string>
```
Comment (david-livefront) [inline app/src/main/res/values-da-rDK/strings.xml:1058]: You do not need to update these other files, just the base one.

The next Crowdin sync will take care of the rest.

Comment (dseverns-livefront) [reply app/src/main/res/values-da-rDK/strings.xml]: So I rolled this back, it does seem to be what is failing the build. Cause the that version of the string no longer exists in the default version. Are we ok to merge, crowdin is a black box to me.

Comment (david-livefront) [reply app/src/main/res/values-da-rDK/strings.xml]: Hmmm, @SaintPatrck do we have a good way of doing this?

Comment (SaintPatrck) [reply app/src/main/res/values-da-rDK/strings.xml]: We can ignore `ExtraTranslation` since Crowdin will be managing this. I opened #4176 to ignore it.

**Review by david-livefront — APPROVED** [review]

**Pull request merged**